### ISSUE TITLE
UTF-8 processing no longer works following 1.1.3 upgrade

### ISSUE THREAD

**Issue by tirdadc** [tilo/smarter_csv#89]
Here's my CSV (saved as a UTF-8 file):

```
"Artist","Track","Album","Label","Year"
"Кино","Мама, мы все сошли с ума","Группа Крови","Moroz Records",1998
```

This works fine with 1.1.1 using:

```
n = SmarterCSV.process(ENV['FILE'])
```

I upgraded to 1.1.3 and it seems to be treating all unicode characters as invalid and stripping them out. I tried setting the `force_utf8` option to `true`, but that didn't help.

I also did try the code snippet below without any success:

```
File.open(filename, "r:bom|utf-8") do |f|
  data = SmarterCSV.process(f);
end
```

**Comment by tilo** [comment]
confirmed! It broke in 1.1.2

Looking into a fix

**Comment by tilo** [comment]
Fixed in 1.1.4
